app.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const app = express();
-const { getTopics, getEndpoints, getArticleId, getArticles } = require('./controllers/get.controller')
+const { getTopics, getEndpoints, getArticleId, getArticles, getCommentsById } = require('./controllers/get.controller')
 
 app.use(express.json());
```

Comment:
Why is this line of code here? We really only should write code we are actively using. At this point, none of our requests use a body, we aren't doing any posts etc.
It's really bad form to not work to the specifications a ticket has and just adding code you *think* will be needed down the line